modules/puppetca/puppetca_puppet_cert.rb

```diff
@@ -39,6 +47,44 @@ def list
       hash.merge(ca_inventory) {|key, h1, h2| h1[:state] == "pending" ?  h1 : h1.merge(h2)}
     end
 
+    def list_puppet_6
```

Comment:
Unfortunately, this does not list `serial`, `not_before` and `not_after` anymore.